app/commands.py

```diff
@@ -344,7 +344,7 @@ async def recent(ctx: Context) -> Optional[str]:
     l = [f"[{s.mode!r}] {s.bmap.embed}", f"{s.acc:.2f}%"]
 
     if s.mods:
-        l.insert(1, f"+{s.mods!r}")
+        l.insert(1, f"+{Mods.filter_invalid_combos(s.mods, s.mode)!r}")
```

Comment:
this is from an existing score, so it shouldn't be able to get into the db in an invalid state

once it's in db, i trust it, so don't need to call this (expensive) func on it

Reply:
sure that makes sense